### PR TITLE
BurgerKing project extended with collection homework

### PR DESCRIPTION
**Pull request by 51stRoR**
update 13.11
BurgerKing project extended with collection homework logic
Notes:
1. As a base List used list of product from Menu object, so all method to show, filter, check is in Menu class
2. Result of filtering dish items stored in additional filtered list in Menu object to be able to work with base menu list in the following flow
3. There is fixed test flow prepared in Main, to change input data(type of filtering, price criteria to check items etc.) - you can change it only manually in the code, there isn't any kind of console menu with inputs for now
4. Last tasks about traversing and collect lowest price per dish category done with 2 additional tmp Lists inside method - this is my first approach to solve this for now, may be redone later in a more better way
5. There are no Map added yet - may be added eventually

### CODE REVIEW

File: src/main/java/BurgerKing/Client.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
+package BurgerKing;
```
Comment (olobanok) [inline src/main/java/BurgerKing/Client.java]: package names should be in lowercase

Comment (51stRoR) [reply src/main/java/BurgerKing/Client.java]: changed to lowercase

**Comment by 51stRoR** [comment]
new request will be done to nkushnir_master with fixed and updated collections task logic

**Comment by 51stRoR** [comment]
check PR #41

**Pull request closed without merging**